### DISCUSSION TITLE
Price of our token doesn't display in phantom wallet, its 4 weeks old now, listed on CMC & CG.

### DISCUSSION THREAD

**Discussion by nightcycle-dev** [phantom/docs#155] [General]
Are there any extra steps that need to be taken to get the price to show, we have nearly 2000 holders and many keep asking this.

$PIF is the ticker and the CA is:
8vRnnknwERunJEEDtEFoxzaxbxGnEY61FPbx8uTVKsUD

Thank you.

**Comment by adamdelphantom** [comment]
Hello @PHinett2,

I checked with some of our price source partners. Would you mind letting me know if this issue is still persistent, please?

**Comment by nightcycle-dev** [comment]
Hi,

It looks like giving them a prod has resolved the issue, the price is 
now showing in the wallet.

Thanks for your help.


------ Original Message ------
From "Adam" ***@***.***>
To "phantom/docs" ***@***.***>
Cc "PHinett2" ***@***.***>; "Mention" 
***@***.***>
Date 19/01/2024 14:29:18
Subject Re: [phantom/docs] Price of our token doesn't display in phantom 
wallet, its 4 weeks old now, listed on CMC & CG. (Discussion #155)
